inputs.md

```diff
@@ -37,6 +37,10 @@ Please do not use the following inputs:
 
```

Comment:
I would remove this file. Most of the things mentioned here are pretty general stuff, so they were moved to the central biowdl page.

Reply:
Done